tests/acceptance/conftest.py

```diff
@@ -276,17 +276,21 @@ def _make_user(
 @pytest.fixture(scope="class")
 def make_project_access_token(
     project,
-) -> Generator[Callable[[AccessLevel, List[str]], ProjectAccessToken], None, None]:
+) -> Generator[
+    Callable[[Project, AccessLevel, List[str]], ProjectAccessToken], None, None
+]:
     token_name_base = get_random_name("user")
     created_tokens: List[ProjectAccessToken] = []
 
     def _make_project_access_token(
-        level: AccessLevel = AccessLevel.DEVELOPER, scopes: List[str] = ["api"]
+        target_project=project,
+        level: AccessLevel = AccessLevel.DEVELOPER,
+        scopes: List[str] = ["api"],
```

Comment:
I expected the `target_project` to be typed like this:

```suggestion
        target_project: Project = project,
        level: AccessLevel = AccessLevel.DEVELOPER,
        scopes: List[str] = ["api"],
```

Even though the tests passes, `mypy` keeps complaining with following errors:

```
tests/acceptance/conftest.py:297: error: Argument 1 to "append" of "list" has incompatible type "RESTObject"; expected "ProjectAccessToken"  [arg-type]
tests/acceptance/conftest.py:298: error: Incompatible return value type (got "RESTObject", expected "ProjectAccessToken")  [return-value]
```

Not really sure why and how to address it other than what I have right now.

Reply:
Frankly I am fed up with constant mypy issues. I don’t recall any case where it helped while I remember many ones where it was just an nuisance. I propose to get rid of this check completely. Any objections, @amimas ?

Reply:
I opened a new issue (https://github.com/gitlabform/gitlabform/issues/586) to continue the discussion.